election-api-scala/app/model/PartyResult.scala

```diff
@@ -8,4 +8,4 @@ case class ApiResponse (error: String, message: String)
 // TODO: this class should hold:
 //  - the overall winner (if there is one)
 //  - the seats that each party wins in Parliament
-case class Scoreboard(winner: String)
\ No newline at end of file
+case class Scoreboard(declared: Int)
```

Comment:
In the context set by the tasks in the [readme](https://github.com/bbc/rms-software-engineering-technical-assessments/blob/main/election-api-scala/tasks.md#task) - "winners are declared" given they have > 325 seats. It doesn't define a requirement for us to show the number of seats that have been counted so far in the election. Should we update the readme to include a test to calculate the number of seats declared too?

Reply:
Commit messages make it really clear what is being changed. If these were consolidation in the PR body it would aid comprehesion even further. 👍